exercises/practice/zebra-puzzle/.meta/config.json

```
@@ -0,0 +1,19 @@
+{
+  "authors": ["rajatnai49"],
+  "files": {
+    "solution": [
+      "zebra_puzzle.cpp",
+      "zebra_puzzle.h"
```

I have to redo the config, until then please manually correct this part to:

```suggestion
      ".meta/zebra_puzzle.cpp",
      ".meta/zebra_puzzle.h"
```

---

> I have to redo the config, until then please manually correct this part to:

Are you sure? I think these two entries for the initial stub are correct, the example solution (`example.h`, `example.cpp`) should be moved to the `.meta/` directory.

---

I think our config.json is not correct, so the files and the part above are not generated according to the rest of the track:

https://github.com/exercism/cpp/blob/dad723ff78e3054f917469895b52c7fb66217f71/config.json#L30